### ISSUE TITLE
peer code review

### ISSUE THREAD

**Issue by andgarzonmal** [waqaskanju/meal-chooser#35]
hello waqas,

Nice job so far, you can try running jour tests that are based on a fetch method making a mock on those functions the mock could be a function that returns something similar to what you want to get when you run the fetch

**Comment by masangana** [comment]
yeah The mock is a good idea. I use it to solve my test issue too
